.vitepress/components/Translator.ts

```diff
@@ -0,0 +1,34 @@
+import {useData} from "vitepress";
+import {messages} from "../theme/enhancements/i18n/i18n";
```

Comment:
the file is missing? But i dont get it why it works ...

Reply:
It's back!

I think it was a problem with merging the commit that reverted the i18n changes. Kind of weird.